components/Episode/Speakers.js

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import {
+  ListOfSpeakers,
+  ItemOfSpeaker,
+  StyledFoldButton,
+} from "./Episode.styled";
+
+export default function Speakers({ speakers }) {
+  const [showListOfSpeakers, setShowListOfSpeakers] = useState(false);
+
+  return showListOfSpeakers === true ? (
```

Comment:
```suggestion
  return showListOfSpeakers ? (
```

Reply:
changed it in the other components as well